src/components/30-organisms/commercial-hero-banner/child.scss

```diff
@@ -13,13 +13,27 @@ axa-commercial-hero-banner {
     overflow-wrap: break-word;
   }
 
-  p {
+  p, .checkmarks {
     @include text(t4);
     @include breakpoint($mediaquery-xs-up) {
       @include text(t3);
     }
   }
 
+  .checkmark {
+    margin-bottom: 5px;
+  }
+
+  .checkmark-icon {
+    color: $color-viridian-green;
+  }
+
+  .checkmark-text {
+    position: relative;
+    top: -5px;
```

Comment:
...you could get rid of this negative positioning. I'm not a fan of negative sizing information, as it's coupled to font and font-size.

Reply:
I did that and this is how it looks like now :
<img width="436" alt="Screenshot 2021-10-05 at 16 26 39" src="https://user-images.githubusercontent.com/37440641/136049293-205df474-000e-4deb-a45b-da81a275c7ff.png">
Properties like justify-content, align-items etc. makes no difference since it is already centralized but in my opinion the text is a bit too much moved to the top, for 1-2 pixels.
Take a look and let me know it it's ok.